tests/foreman/installer/test_installer.py

```diff
@@ -1624,36 +402,29 @@ def test_positive_check_installer_hammer_ping(target_sat):
             assert 'ok' in line
 
 
-@pytest.mark.e2e
+@pytest.mark.stream
 @pytest.mark.upgrade
 @pytest.mark.tier3
-@pytest.mark.parametrize('filter', ['params', 'sections'])
-def test_installer_options_and_sections(filter):
-    """Look for changes on installer sections and options/flags
+@pytest.mark.build_sanity
+def test_installer_modules_check(target_sat):
+    """Look for changes in installer modules
 
     :id: a51d3b9f-f347-4a96-a31a-770349db08c7
```

Comment:
Yes easier to maintain. The other functionality of this test where it checks if params have changed will be moved upstream at a PR level so devs know if the change is expected or not. They won't have to wait for us.

Regarding test id, I'd leave it the same since it's still covering the modules / sections check from before. It's just an improved version.